src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -408,20 +265,7 @@ public SolidColorBrush PortBackgroundColor
             }
         }
 
-        /// <summary>
-        /// Sets the color of the use levels popup in the input port context menu.
-        /// This changes when the Keep List Structure option is activated and the port
-        /// is connected, upon which it turns blue.
-        /// </summary>
-        public SolidColorBrush UseLevelsMenuColor
-        {
-            get
-            {
-                return ShouldKeepListStructure && _port.IsConnected
-                    ? new SolidColorBrush(Color.FromArgb(255, 60, 60, 60))
-                    : new SolidColorBrush(Color.FromArgb(255, 83, 83, 83));
-            }
-        }
```

Comment:
And this?

Reply:
Same again here, I added this in very recently